flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorTest.java

```diff
@@ -983,6 +985,139 @@ public void testSlotAcceptance() throws Exception {
         }
     }
 
+    private enum ResponseOrder {
+        ACCEPT_THEN_REJECT,
```

Comment:
Should we also test explicitly that it is possible to free an inactive slot?

Reply:
I think that is a good idea; I did wonder a bit what the behavior in that case was.